runtime/neumann/src/xcm_config.rs

```diff
@@ -166,19 +173,70 @@ pub type Barrier = DenyThenTry<
 	),
 >;
 
+/// Based on Kusama values.
+pub fn roc_per_second() -> u128 {
+	CurrencyId::ROC.cent() * 16
+}
+
+/// Based on Turing values.
+pub fn neu_per_second() -> u128 {
+	let neu_equivalent = convert_to_token(CurrencyId::ROC, CurrencyId::Native, roc_per_second());
+	neu_equivalent * 260
+}
+
+parameter_types! {
+	pub NeuPerSecond: (AssetId, u128) = (
+		MultiLocation::new(
+			1,
+			X1(Parachain(u32::from(ParachainInfo::parachain_id()))),
+		).into(),
+		neu_per_second()
+	);
+
+	pub NeuCanonicalPerSecond: (AssetId, u128) = (
+		MultiLocation::new(
+			0,
+			Here,
+		).into(),
+		neu_per_second()
+	);
+
+	pub RocPerSecond: (AssetId, u128) = (MultiLocation::parent().into(), roc_per_second());
+}
+
+pub struct ToTreasury;
+impl TakeRevenue for ToTreasury {
+    fn take_revenue(revenue: MultiAsset) {
+        if let MultiAsset {
+            id: AssetId::Concrete(id),
+            fun: Fungibility::Fungible(amount),
+        } = revenue
+        {
+            if let Some(currency_id) = CurrencyIdConvert::convert(id) {
+                // Ensure TreasuryAccount have ed requirement for native asset, but don't need
+				// ed requirement for cross-chain asset because it's one of whitelist accounts.
+				// Ignore the result.
+				let _ = Currencies::deposit(currency_id, &TreasuryAccount::get(), amount);
```

Comment:
nit: formatting

Reply:
believe it or not the format looks correct in my IDE, so not sure what is going on here